Sming/Libraries/ModbusMaster/samples/generic/component.mk

```diff
@@ -0,0 +1,3 @@
+ARDUINO_LIBRARIES := ModbusMaster
+DISABLE_SPIFFS = 1
+COMPONENT_CXXFLAGS := -DMB_RESPONSE_TIMEOUT_MS=300
```

Comment:
As per. above you can replace this line with:

```
MB_RESPONSE_TIMEOUT := 300
```

This will set the default value for the sample application. You can override this using `make MB_RESPONSE_TIMEOUT=1000`. Then `make config-clean` will revert to this project default.

If you don't specify a value here, then it will use the default from the library.